packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -163,6 +190,16 @@ const patternSource = computed(() => {
     &::selection {
       background: var(--selection-color);
     }
+
+    #{get-autofill-selectors("&")} {
+      background-color: transparent;
+      -webkit-text-fill-color: var(--onyx-color-text-icons-neutral-intense);
+
+      // many browsers use "!important" to set the autofill background so we need this
+      // transition workaround to make the background transparent
+      $one-year-in-seconds: 31536000s;
+      transition: background-color $one-year-in-seconds;
```

Comment:
The styles are not done by the user, they are done by the browser / user-agent, see [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:autofill).
![image](https://github.com/SchwarzIT/onyx/assets/67898185/6b55d784-055a-441e-9da9-03281c14c38a)

I would assume they use !important to ensure that autofill styles are applied when the input has custom styles that do not take autofill into account like we do.

So if we want to implement our own styles for the autofill state, we need this kind of workaround to prevent the default browser styles

Reply:
Do you know which browsers are affected?

Reply:
From a quick test:
- Arc
- Brave
- Firefox

So I guess basically every browser

Reply:
We agreed to use the workaround for `:autofill` as this is an issue in every browser.